en/CommandLine.md

```diff
@@ -1,77 +1,174 @@
+(or `--automaticallySetFileLinks`)
 
-## Export file: -o filename\[,format\]
+Automatically set file links.
 
-This option causes JabRef to save or export a file imported or loaded by the same command line. If a file is imported using the `-i` option, that database will be exported. Otherwise, the *last* file specified (and successfully loaded) without the `-i` option will be exported.
 
-If only a filename is specified, it will be saved in BibTeX format. If the filename is followed by a comma and an export format, the given export filter will be used. A custom export filter can be used in this way, and will be preferred if the export name matches both a custom and a standard export filter.
+### Regenerate keys: `-g` 
+(or `--generateBibtexKeys`)
 
-Use the `-h` option to get a list of availiable export formats.
+Regenerate all keys for the entries of a BibTeX file.
 
-If the `-n` option has not been invoked, any export operation will be performed before the JabRef window is opened, and the imported database will show up in the window.
 
-The -o option can be specified only once, and for one file only.
+### Export preferences: `-x filename`
+(or `--prexp filename`)
 
-## Export preferences: -x filename
+Export user preferences to an XML file.
+After exporting, JabRef will start normally.
 
-Using this option, you can have JabRef export all user preferences to an XML file. After exporting, JabRef will start normally.
 
-## Import preferences: -p filename
+### Import preferences: `-p filename`
+(or `--primp filename`)
 
-This option causes JabRef to import user preferences exported using the `-x` option. After importing, JabRef will start normally.
+Import user preferences from an XML file (exported using the `-x` option, or through the GUI).
+After importing, JabRef will start normally.
 
-## Export matching entries: -m \[field=\]searchTerm,outputFile\[,exportFormat\]
 
-JabRef saves all the database entries matching to a given search term in a new file. The export file's format can be chosen, the default format is html-table (with abstract and bibtex, provided by tablerefsabsbib).
+### Reset preferences: `-d key`
+(or `--prdef key`)
 
-Call: `java -jar JabRef.jar -m [field=]searchTerm,outputFile[,exportFormat] -n true inputFile`
+Reset preferences (key1, key2,..., or `all`).
 
-For information referring to the search function see the documentation for 'advanced search'. In addition it is also possible to search for entries within a timeframe instead of only searching for entries of a certain year.
 
-Please note that search terms containing blanks need to be bracketed by quotation marks.
+### No files at startup: `-b`
+(or `--blank`)
+Do not open any files at startup
 
-Examples
 
--   `Year=2005`
--   `title|keywords=Optimization`
--   `(author=bock or title|keywords="computer methods")and not(author=sager)`
--   `Year=1989-2005`
+### Version: `-v`
+(or `--version`)
 
-## Export only used items: -a filename\[.aux\],newBibFile\[.bib\]
```

Comment:
Doesn't that work anymore?

I have to take time to really review this PR. Hope, I find time next week.